_docs/useful-links.md

```diff
@@ -10,7 +10,7 @@ summary: A list of links for Data Providers
 ---
 
 <!--
-> [Wiki](https://github.com/kerfoot/ioosngdac/wiki) ▸ **Data Provider Links**
+> [Wiki](https://github.com/kerfoot/glider-dac/wiki) ▸ **Data Provider Links**
```

Comment:
I see this is commented out, but given Derrick's guidance that we are not to use the repo Wiki, we'll need to remove this sentence entirely.